src/StudentDashboard/StudentDashboard.scss

```diff
@@ -0,0 +1,73 @@
+.student-body {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  background-color: #56B4E9;
+}
+
+.student-dash {
+  display: flex;
+  height: 100vh;
+  width: 100vw;
+  align-items: center;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  // text-align: center;
+}
+
+.reading {
+  width: 40%;
+  height: 25vh;
+  padding: 10px;
+  font-size: 2.5vh;
+  background-color: white;
+  border: 5px outset #009E73;
+  border-radius: 5px;
+  box-shadow: 3px 2px 6px;
+  overflow-y: scroll;
+}
+
+.question {
+  width: 90%;
+  height: 50vh;
+  text-align: center;
+  background-color: #F0E442;
+  border-radius: 5px;
+  box-shadow: 3px 2px 6px;
+  border: 5px outset #E69F00; 
+}
+
+.animation {
+  width: 40%;
+  height: 25vh;
+  padding: 10px;
+  background-color: white;
+  border: 5px outset #D55E00;
+  border-radius: 5px;
+  box-shadow: 3px 2px 6px;
+  background-image: url('https://i.pinimg.com/originals/f3/64/b7/f364b7d0a49566891789d291b8b2e53f.png');
+  background-size: contain; 
+}
+
+header {
+  margin: 0;
+  width: 100vw;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.app-name {
+  margin-left: 5vw;
+  font-size: 24px;
+}
+
+h4 {
```

Comment:
Do you see conflicts with the other h4?  Just curious.

Reply:
I dont think so, if you gave yours a classname it should override it